File: 3477-fruits-into-baskets-ii/3477-fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        n = len(fruits)
        count=0
        used=[False for i in range(n)]

        for i in range(n):
            placed = False
            for j in range(n):
                if(fruits[i]<=baskets[j] and used[j]==False):
                    used[j]=True
                    placed=True
                    break
            if(placed==False):
                count+=1

        return count

        

File: 3477-fruits-into-baskets-ii/3477-fruits-into-baskets-ii.cpp
class Solution {
public:
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int n = fruits.size();
        int count = 0;
        vector<bool> used(n, false);

        for(int i=0; i<n; i++){
            bool placed = false;
            for(int j =0; j<n; j++){
                if(fruits[i]<=baskets[j] && used[j]==false){
                    used[j] = true;
                    placed = true;
                    break;
                }
            }
            if(placed!=true){
                count++;
            }
        }
        return count;
        
    }
};